bindgen/codegen/mod.rs

```diff
@@ -4391,7 +4397,23 @@ impl TryToRustTy for Type {
                 {
                     Ok(ty)
                 } else {
-                    Ok(ty.to_ptr(is_const))
+                    let ty_ptr = ty.to_ptr(is_const);
```

Comment:
So, I guess that means that this should be something like: "if reference, then `Reference<#ty>` else `ty.to_ptr(is_const)`"?